community.md

```diff
@@ -88,4 +88,5 @@ Below is a list of projects and resources from the Blazor community (links in ea
 ## Other
 
 * [Bionic](https://github.com/bmsantos/bionic) &ndash; An Ionic CLI clone for Blazor projects.
+* [A Nicer Looking Blazor Loading Page](http://lightswitchhelpwebsite.com/Blog/tabid/61/EntryId/4315/A-Nicer-Looking-Blazor-Loading-Page.aspx) &ndash; Customize the Blazor loading page by using Tobias Ahlin's SpinKit.
```

Comment:
I think this would be better as a sample actually. The main link would point to the sample on GitHub and then we would link to the blog post in parens:

```markdown
[A Nicer Looking Blazor Loading Page](https://github.com/ADefWebserver/BlazorSpinKitLoadingPage) ([blog post](http://lightswitchhelpwebsite.com/Blog/tabid/61/EntryId/4315/A-Nicer-Looking-Blazor-Loading-Page.aspx)) &ndash; Customize the Blazor loading page by using [SpinKit](https://github.com/tobiasahlin/SpinKit). 
```